WooCommerce/Classes/Authentication/Navigation Exceptions/RoleErrorViewController.swift

```diff
@@ -88,6 +97,14 @@ class RoleErrorViewController: UIViewController {
 
     func configurePrimaryActionButton() {
         primaryActionButton.applyPrimaryButtonStyle()
+
+        // Since the buttons are vertically stacked, override the font
+        // on iPhone interface idioms so we get a maximum font size to give
+        // space for the content above the button container.
+        if WPDeviceIdentification.isiPhone() {
+            primaryActionButton.titleLabel?.font = StyleManager.headlineSemiBold
+        }
```

Comment:
I'm not sure if this is a good alternative since if users use a gigantic font size is because they need it 😅 

Look the feedback completion screen is handling this scenario.(I modified it to have a multiple line button)

<img width="385" alt="Screen Shot 2021-07-15 at 4 23 02 PM" src="https://user-images.githubusercontent.com/562080/125860089-98217c6a-786f-4201-8f0c-3df102f3e0ba.png">

Reply:
@Ecarrion 

Ah, I see. Thanks for the input! I'd like to clarify some assumptions that I've had, that led me to design it this way:
1. With increased text sizes, the Help menu is always visible (since it's located on the navigation bar), while the action buttons would be hidden below-the-fold. From the user experience perspective, **I assumed that preserving the action buttons above-the-fold would help users be more aware of what actions they could take**. However, in iPhone landscape mode, the buttons are taking too much of the viewport when the text size are allowed to be increased naturally.
2. The font style with maximum font size is available to use in `StyleManager`. I assumed that everything there is a11y-friendly (at least to some acceptable degree). This raises a question though: does this mean that `StyleManager.headlineSemiBold` (or any font presets in `StyleManager` that have max font size) should not be used? 🤔 
3. Taking into account the interfaces around iOS when maximum a11y text size is selected, I noticed that not every interface component is inflated to a gigantic font size. For example: navigation bar buttons. This led to think that it's _okay_ to use a smaller size, as long as it's not the smallest font size displayed within that accessibility setting.

Also, thank you for sharing your reference! 👍 With the latest commit, I've modified the layout to be entirely scrollable, and the max font size on the action buttons are removed. Here's how it looks now:

Portrait | Landscape
--|--
![Simulator Screen Shot - iPhone 8 - 2021-07-17 at 22 38 17](https://user-images.githubusercontent.com/1299411/126042289-68ccd5f9-2a33-4b47-a966-76d5368eb93b.png) | ![Simulator Screen Shot - iPhone 8 - 2021-07-17 at 22 20 10](https://user-images.githubusercontent.com/1299411/126042292-8ebff29f-98ea-4fe2-b193-1ad056280349.png)

Let me know what you think!